component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -1266,5 +1280,12 @@ protected void triggerNotification(String userName, String tenantDomain, String
         }
     }
 
+    private String getHttpErrorResponseCode(String errorMsg) {
 
-}
\ No newline at end of file
+        String errorCode = errorMsg;
+        if (StringUtils.contains(errorCode, ":")) {
+            errorCode = errorCode.split(":")[0];
+        }
+        return errorCode.trim();
```

Comment:
check whether errorCode is null before calling trim(). Check whether there's a trim function in StringUtils then we can directly use that

Reply:
Null check is already handled here https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/81/files/60fd47f50a17eca7eded2ab24babaec65530dbd7#diff-862199ef8d38f6df18c5f44aeeb2e115R575

Reply:
StringUtils is used